charts/catalog/templates/rbac.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
+kind: List
```

Comment:
@liggitt gave it to me like this. I dunno if there's any general preference or a behavioral difference between a list and a stream of objects.